bin/preload-worker.php

```
@@ -0,0 +1,40 @@
+}
+
+[, $appName, $context, $appDir, $writeDir] = $argv;
+
+// Buffer before the first include: a deprecation printed by any autoloaded file would mark
+// the headers as sent, and the responder this boot must record skips its work when they are.
+// Failures go to STDERR, which no buffer touches.
+ob_start();
+require $appDir . '/vendor/autoload.php';
+
+try {
+    (new PreloadRecorder())($appName, $context, $appDir, $writeDir === '' ? null : $writeDir);
```

PHPStan (2.2.7)

[error] Parameter `#1` $appName of callable BEAR\Package\Compiler\PreloadRecorder expects non-empty-string, string given.


(argument.type)